apps/rush-lib/src/logic/buildCache/ProjectBuildCache.ts

```diff
@@ -484,6 +484,9 @@ export class ProjectBuildCache {
 
     const sortedProjectStates: string[] = projectStates.sort();
     const hash: crypto.Hash = crypto.createHash('sha1');
+    // This value is used to force cache bust when the build cache algorithm changes
+    hash.update(`${RushConstants.buildCacheVersion}`);
+    hash.update(RushConstants.hashDelimiter);
```

Comment:
```suggestion
    if (RushConstants.buildCacheVersion !== 1) {
      // This value is used to force cache bust when the build cache algorithm changes
      hash.update(`${RushConstants.buildCacheVersion}`);
      hash.update(RushConstants.hashDelimiter);
    }
```

As to not break existing caches - unless that's the idea.

Reply:
@iclanton This change is intended to break existing caches to ensure that #3211 fully replaces any cache entries from the preceding release.